osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -27,7 +28,7 @@ namespace osu.Game.Screens.Play
     /// <remarks>
     /// This is intended to be used as a single controller for gameplay, or as a reference source for other <see cref="GameplayClockContainer"/>s.
     /// </remarks>
-    public class MasterGameplayClockContainer : GameplayClockContainer
+    public class MasterGameplayClockContainer : GameplayClockContainer, IBeatSyncProvider
```

Comment:
Is there a reason why both `Player` and `MasterGameplayClockContainer` need to implement this? I'm guessing it's due to tests or something of the sort?

Reply:
Actually not required. I first implemented here but didn't check back after the `MasterGameplayClockContainer` implementation to confirm whether it was still required.